docs/references/content-as-code.mdx

```diff
@@ -26,10 +26,74 @@ For example:
 - I run `lightdash download` again 
 - The changes I made to `emea-revenue-per-month.yml` will be overwritten by the latest chart version downloaded from the Lightdash application 
 
+### Use `lightdash download -c` or `lightdash download --charts` to select specific charts
+
+If you only want to download specific charts to manage as code, you can use the chart selector in the download command. 
+
+For example, if I only wanted to download a specific saved chart as code, I would run the command:
+
+```bash
+lightdash download -c https://app.lightdash.cloud/the-url-to-my-saved-chart
+```
+
+You can use the chart's SLUG, UUID, or the URL to the saved chart to select the chart.
+
+### Use `lightdash download -d` or `lightdash download --dashboards` to select specific dashboards
+
+For example, if I only wanted to download a specific dashboard as code, I would run the command:
+
+```bash
+lightdash download -d https://app.lightdash.cloud/the-url-to-my-dashboard
+```
+
+You can use the dashboard's SLUG, UUID, or the URL to the dashboard to select the dashboard.
+
+#### To select multiple charts or dashboards, add a space between the items
+
+For example, this command would select two charts to download:
+
+```bash
+lightdash download -c https://app.lightdash.cloud/the-url-to-my-first-saved-chart https://app.lightdash.cloud/the-url-to-my-second-saved-chart
+```
+
+You can combine charts and dashboards selection in a single command. For example, this command would download a chart and a dashboard:
+
+```bash
+lightdash download -c https://app.lightdash.cloud/the-url-to-my-first-saved-chart -d https://app.lightdash.cloud/the-url-to-my-dashboard
+```
+
 ## `lightdash upload` updates any content as code to your project
 
 From the [Lightdash CLI](/references/lightdash-cli), you can use the command `lightdash upload` to upload any changes you've made to your charts or dashboards as code. To upload new charts that you've created as code to your Lightdash project, you need to run `lightdash upload --force`
 
+### Use `lightdash upload -c` or `lightdash upload --charts` to select specific charts
+
+For example, if I only wanted to upload a specific saved chart as code, I would run the command:
```

Comment:
remember these charts will only be uploaded if the change was updated locally. I'm not sure if that's worth mentioning.